Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -84,14 +96,14 @@ class HistoryPanel: SiteTableViewController, HomePanel {
     func notificationReceived(notification: NSNotification) {
         switch notification.name {
         case NotificationFirefoxAccountChanged, NotificationPrivateDataClearedHistory:
-            resyncHistory()
+            tableViewController.resyncHistory()
             break
         case NotificationDynamicFontChanged:
-            if emptyStateOverlayView.superview != nil {
-                emptyStateOverlayView.removeFromSuperview()
+            if tableViewController.emptyStateOverlayView.superview != nil {
+                tableViewController.emptyStateOverlayView.removeFromSuperview()
             }
-            emptyStateOverlayView = createEmptyStateOverview()
-            resyncHistory()
+            tableViewController.emptyStateOverlayView = tableViewController.createEmptyStateOverlayView()
+            tableViewController.resyncHistory()
```

Comment:
Nit: All of the logic inside this notification selector accesses attributes inside the `tableViewController`. Why not handle the notification inside `HistoryPanelViewController` instead? Doesn't seem this VC needs to know about the notification.